CMakeLists.txt

```diff
@@ -1341,6 +1343,11 @@ if(WITH_JPEG OR HAVE_JPEG)
     status("    JPEG:" NO)
   elseif(BUILD_JPEG)
     status("    JPEG:" "build-${JPEG_LIBRARY} (ver ${JPEG_LIB_VERSION})")
+    if(WITH_LIBJPEG_TURBO_SIMD)
+      status("        SIMD Support:" "YES")
+    else()
+      status("        SIMD Support:" "NO")
+    endif()
```

Comment:
We should distinguish a request from the user and the actual "final" value (which depends on the platform (e.g. MSA is not handled yet), compiler, etc)

Also there is still "libjpeg" (without "-turbo")

Reply:
"libjpeg" without  -turbo will report `"SIMD Support: "NO"` by default. And i think this is not a false promise. I don`t think that i should worry about that. And libjpeg is deprecated according to [docs](https://github.com/opencv/opencv/tree/4.x/3rdparty) at 4.x.